### PR TITLE
Store future self dialog

### PR DESCRIPTION
**Pull request by bscheltinga**
fixes #180

### CODE REVIEW

File: Rasa_Bot/actions/actions.py [hunk 1]
```diff
@@ -28,6 +27,25 @@
...
 
 
+class DialogQuestions(Enum):
+    FUTURE_SELF_SMOKER_WORDS = 1
+    FUTURE_SELF_SMOKER_WHY = 2
+    FUTURE_SELF_MOVER_WORDS = 3
+    FUTURE_SELF_MOVER_WHY = 4
+
+
+def store_dialog_answer_to_db(user_id, answer, question: DialogQuestions):
```
Comment (svenvanderburg) [inline Rasa_Bot/actions/actions.py:37]: I really like the interface of this function!

File: Rasa_Bot/actions/actions.py [hunk 2]
```diff
@@ -28,6 +27,25 @@
 NICEDAY_API_ENDPOINT = os.getenv('NICEDAY_API_ENDPOINT')
 
 
+class DialogQuestions(Enum):
+    FUTURE_SELF_SMOKER_WORDS = 1
+    FUTURE_SELF_SMOKER_WHY = 2
+    FUTURE_SELF_MOVER_WORDS = 3
+    FUTURE_SELF_MOVER_WHY = 4
```
Comment (svenvanderburg) [inline Rasa_Bot/actions/actions.py:34]: Maybe you can add a tiny bit of documentation here so it is easier to remember what questions these refer to. Could be as simple as adding a comment:
`# Why did you pick these words for moving?`

File: Rasa_Bot/domain.yml [hunk 3]
```diff
@@ -391,7 +391,7 @@ actions:
 - validate_pa_evaluation_form
 - action_utter_pa_evaluation_form_filled
 - action_store_pa_evaluation
-### Future self dialog
+- action_store_smoker_words
```
Comment (svenvanderburg) [inline Rasa_Bot/domain.yml:394]: Would be nice to keep the comment that these actions are for future self dialog

**Review by svenvanderburg — COMMENTED** [review]
Nice work @bscheltinga ! ⭐  See my comments in the code. Also:
* Please also store the other 3 answers
* The linter is failing, there is one line that is too long: 
image
* You committed 2 new model files, please keep only the most recent one and remove the current model file.

If you want I can take another look before you merge, but feel free to merge if you feel confident.

File: Rasa_Bot/domain.yml (hunk 3 above)
Comment (bscheltinga) [reply Rasa_Bot/domain.yml:394]: Yes, good idea

**Pull request merged**